cmd/version.go

```diff
@@ -3,28 +3,68 @@ package cmd
 import (
 	"fmt"
 
+	"github.com/exercism/cli/cli"
 	"github.com/spf13/cobra"
 )
 
 // Version is the version of the current build.
 // It follows semantic versioning.
 const Version = "3.0.0-alpha.1"
 
+// checkLatest flag for version command.
+var checkLatest bool
+
 // versionCmd outputs the version of the CLI.
 var versionCmd = &cobra.Command{
 	Use:     "version",
 	Aliases: []string{"v"},
 	Short:   "Version outputs the version of CLI.",
 	Long: `Version outputs the version of the exercism binary that is in use.
 
-To see if there is a more recent version available, call the command with the
+To check for the latest available version, call the command with the
 --latest flag.
 	`,
+
 	Run: func(cmd *cobra.Command, args []string) {
-		fmt.Println(Version)
+		fmt.Println(currentVersion())
+
+		if checkLatest {
+			c := cli.New(Version)
+			l, err := checkForUpdate(c)
+			BailOnError(err)
+
+			fmt.Println(l)
+		}
+
+		return
 	},
 }
 
+// currentVersion returns a formatted version string for the Exercism CLI.
+func currentVersion() string {
+	return fmt.Sprintf("exercism version %s", Version)
+}
+
+// checkForUpdate returns a string indicating if the current running version of the Exercism CLI is at the latest release.
+// For CLI versions found to be out of date a string with upgrade instructions is included in the return.
```

Comment:
This is a total nitpick, but it's something I think about a lot, so I wanted to bring it up.

I try to avoid talking about data types in the comments, unless it's information that is hard to pick up from reading the code. Here my suggestion would be:

> checkForUpdate verifies if the CLI is running the latest release.
> If the client is out of date, the function returns upgrade instructions.


I wrote a bit about a related topic a while back: http://whipperstacker.com/2015/10/14/idiomatic-doc-comments-document-your-function-not-your-function-signature/